server/liveblog/posts/utils.py

```diff
@@ -97,6 +99,11 @@ def calculate_post_type(post, items=None):
                 post_items_type = "{}-{}".format(
                     post_items_type, items[0]["meta"]["provider_name"].lower()
                 )
+        elif (
+            items[0].get("item_type", "").lower() == "poll"
+            and items[1].get("item_type", "").lower() == "text"
```

Comment:
It seems to me that `items[0].get("item_type", "").lower()` and `items[1].get("item_type", "").lower()` is being used multiple times in the if/else block. Maybe we should assign it to a variable to improve readability?

Reply:
Also, we should cover this scenario of the `poll` item type in the tests.